app/models/broken_window/measured_metric.rb

```diff
@@ -20,6 +20,17 @@ def status
       @status ||= calculate_status
     end
 
+    def unique_daily_measurements
+      @unique_daily_measurements ||= measurements
+        .group_by{ |measurement| measurement.created_at.to_date }
+        .map{ |_, measurements_for_date| measurements_for_date.max_by(&:created_at) }
```

Comment:
I don't think max will work. pretty sure there are occasions where it should be min